pom.xml

```diff
@@ -13,39 +13,22 @@
  * information: "Portions copyright [year] [name of copyright owner]".
  *
  * Copyright 2017 ForgeRock AS.
- * 
+ *
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+    <parent>
+        <artifactId>openam-auth-trees</artifactId>
+        <groupId>org.forgerock.am</groupId>
+        <version>6.5.0</version>
+    </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>operating-system-checker-auth-tree-node</artifactId>
-    <groupId>org.forgerock.am</groupId>
-    <version>6.0.0-SNAPSHOT</version>
-    
+    <artifactId>os-checker-node</artifactId>
```

Comment:
Renaming the artifact may be confusing for end-users, would recommend leaving it as is (so that it matches the github repository name).